### DISCUSSION TITLE
thanks

### DISCUSSION THREAD

**Discussion by BentleyWood** [angband/angband#5988] [General]
I just want to say thanks guys. Thanks for making angband way back when, because it was awesome when it came out, and I loved it - and thanks for sticking around and maintaining it. I streamed it a couple of weeks ago. I can't say the audience went wild, but they definitely dropped lots of nostalgia in the chat. They don't make them like they used to!

**Comment by NickMcConnell** [comment]
I've only come in relatively recently, it's a torch that's been passed many times.  But thanks on behalf of all the contributors 😄
